packages/e2e-tests/specs/editor/plugins/cpt-locking.test.js

```diff
@@ -85,6 +86,18 @@ describe( 'cpt locking', () => {
 			expect( await getEditedPostContent() ).toMatchSnapshot();
 		} );
 
+		it( 'should insert line breaks when using enter and shift-enter', async () => {
+			await page.click(
+				'.block-editor-block-list__block[data-type="core/paragraph"]'
+			);
+			await page.keyboard.type( 'First line' );
+			await pressKeyTimes( 'Enter', 1 );
```

Comment:
I don't think that this should work for Enter too. Only Shift+Enter is kind of known for inserting a soft line break.

Reply:
I'd disagree lots of my clients are not aware of that and silently treating Enter as a soft line break is a much nicer UX if you have an entire post type locked. This is also how it used to work before the regression.

Reply:
Not sure if this is the right place to post, but I would like to agree with @ocean90 and have this reverted to only apply to Shift+Enter. I'm not sure what pressing Enter did in the past, but what it should do is exit the block, like it does when pushing Enter after any other paragraph block.

Reply:
Just as a use case, I'm setting up a simple block which allows someone to enter one paragraph, and display an icon beside it (using the ACF plugin with InnerBlocks locked to one paragraph, and an image field).

This works fine, but I keep hitting enter after that paragraph so that I can type a normal paragraph, and instead end up with a line break inside the original block.